pydy/tests/test_utils.py

```diff
@@ -18,3 +22,61 @@ def test_sympy_equal_to_or_newer_than():
     if parse_version(SETUPTOOLS_VERSION) >= parse_version('8.0'):
         with assert_raises(ValueError):
             sympy_equal_to_or_newer_than('0.7.7', '0.7.6-git')
+
+def test_mass():
+
+    mass = Mass()
+    assert mass.symbol == Symbol('m')
+    assert mass.value == 1
```

Comment:
Keep in mind that what you build should work with only symbolics provided. Ideally a user can fully describe the bodies/particles, the joints, and the forces with only symbolic parameters and it will ultimately spit out symbolic equations of motion. We currently have a dividing line between symbolics and numerics. All of our EoM generation methods should work in a purely symbolic way.